src/main/java/sk/hfa/startup/DummyIndividualProjectRunner.java

```diff
@@ -49,7 +49,7 @@ public void run(String... args) throws IOException {
             ts3.setText("Rodinný dom je navrhnutý z keramických tvárnic, ktoré možno zameniť za pórobetón. " +
                     "Stropy sú monolitické betónové. Vykurovanie je podlahové.");
             Project commonProject = new CommonProject();
-            commonProject.setCategory(Category.COMMON);
+            commonProject.setCategory(Category.INTERIOR_DESIGN);
```

Comment:
Prečo je použitá kategória pre interiérový dizajn?

Reply:
no idea, uz si nepamatam preco som si to nastavoval :D :hankey: